### DISCUSSION TITLE
Feature support / request

### DISCUSSION THREAD

**Discussion by Dimi-V** [nhn/tui.calendar#961] [Q&A] [answered]
Hello,

We started implementing tui.calendar and we are facing some limitations.
Is there a possibility that the following items are somehow possible or will be included in the next update:

- flexible day view, for instance Monday until Friday + Monday and Tuesday.
- an option to hide certain days (which can be predetermined)
- internally divide each day into sub columns, ex. you have three employees (who each have their internal column), each can only perform one task at a given moment. The output only shows the availability in one column, ex. if one of these employees is available, you can show it as "free" in one column externally.
- set a predetermined slot time for a given calendar, ex. "cleaning desks" calendar has fixed 1h slot
- block certain time slots
- make time slots anonymous for people (the lock option hides info until you click the appointment

Thank you

**Comment by adhrinae** [comment, marked as answer]
I cannot guarantee what will be included in the next update, however, there are some common requests as you mentioned. Anyway, It will take some time.

But I'll keep in mind your request and try to ship more and better features later.
